lib/rack/test.rb

```diff
@@ -262,6 +262,12 @@ def env_for(uri, env)
         Rack::MockRequest.env_for(uri.to_s, env)
       end
 
+      def multipart_content_type(env)
+        requested_content_type = env['CONTENT_TYPE']
+        return requested_content_type if requested_content_type.start_with?('multipart/')
```

Comment:
I would rather skip the return statement and just make it an if-else statement for simplicity.
```suggestion
        if requested_content_type.start_with?('multipart/')
          requested_content_type
        else
          'multipart/form-data'
        end
```